### ISSUE TITLE
Cannot use mean strategy with non-numeric data

### ISSUE THREAD

**Issue by zhenlingcn** [openml/automlbenchmark#355]
I use the released version of this package (v1.6), it seems that there is a bug in the data imputation procedure. After running the experiment, it appears that it cannot produce correct results for the "car" dataset. By checking the training log, I find that the reason should be that there is some non-numerical data in the training data, while the imputation function can only deal with numerical data. How can I solve that problem? Thanks!
![image](https://user-images.githubusercontent.com/18747119/126452643-45575daf-16f5-437b-bdeb-e69e3726dcb5.png)

**Comment by sebhrusen** [comment]
Hi @zhenlingcn, sorry we've stopped making releases for about a year as we started to prepare `v2` since (being released soon) and many issues + changes occured on `master` since from which we'll release `v2`.
Do you mind testing with `master` instead until then? It's relatively stable (actually it's a matter of days before we make the release and merge it to `stable` branch.

**Comment by zhenlingcn** [comment]
Great, I am glad to hear that a new version of this package will be released. I will wait for the release of the stable version because I found there might exist some bugs in the current master branch, based on a trail made several days ago. By the way, are there any new baseline results that will be published with the release of the new package? I hope to compare my algorithm with state-of-the-art algorithms in my paper, but I don't have enough computational resources to completely run all experiments.

**Comment by sebhrusen** [comment]
> I found there might exist some bugs in the current master branch

if you found untracked bugs, please create a detailed issue. Several of us are regularly using the `master` branch during our own framework development, and I'm not aware of any major issue currently that would justify to use an old version like `v1.6` vs. `master`.

> are there any new baseline results that will be published with the release of the new package?

we will run a new baseline as soon as v2 is released.

**Comment by zhenlingcn** [comment]
Great, I am glad to hear that a new baseline will be published. As for the untracked bug, I find it is related to some commented codes. More specifically, I use "ThreadPoolExecutorJobRunner" in my code rather than "MultiThreadingJobRunner". However, the worker function belonging to that runner is outdated. Consequently, I encountered an error during the task execution.
```python
def worker(job):
    result, duration = job.start()
    job.done()
    return Namespace(name=job.name, result=result, duration=duration)
```
I successfully solved this problem by replacing that function with the following code.
```python
def worker(job):
    return job.start()
```
